test/lint/import-resolver.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+const resolverNode = require( 'eslint-import-resolver-node' );
```

Comment:
`eslint-import-resolver-node` should be listed as a dev dependency in the main `package.json` file.